assets/_sass/components/_audio-player.scss

```diff
@@ -0,0 +1,91 @@
+.audio-player {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 1rem 0;
+
+  @include breakpoint('medium') {
+    flex-wrap: nowrap;
+  }
+
+  .audio-control {
+    max-width: 10%;
+    margin: 0 40px 0 0;
+    @include breakpoint('medium') {
+      order: -1;
+      -webkit-box-ordinal-group: -1;
+    }
+  }
+
+  .action {
+    border: none;
+    padding: 0;
+    background-color: transparent;
+  }
+
+  .audio-control,
+  .secondary-info {
+    display: flex;
+
+    align-items: center;
+  }
+
+  .action,
+  .duration {
+    @extend %labelsmall;
+  }
+
+  .action {
+    margin: 0 40px 0 10px;
+  }
+
+  .duration {
+    margin: 0 10px 0 25px;
+  }
+
+  .progress-bar {
+    position: relative;
+    background-color: $color_audio-player-progress-bar;
+    width: 100%;
+    height: 8px;
+    margin-bottom: 8px;
+
+    @include breakpoint('medium') {
+      // width: calc(100% - 225px);
```

Comment:
Let's remove this comment since we don't need it anymore

Reply:
removed line 56 on _audio-player.scss